auto_submit/lib/requests/github_webhook.dart

```diff
@@ -33,9 +42,14 @@ class GithubWebhook extends RequestHandler {
     hasAutosubmit = pullRequest.labels!.any((label) => label.name == 'autosubmit');
 
     if (hasAutosubmit) {
-      // TODO(kristinbi): Check if PR can be submitted. https://github.com/flutter/flutter/issues/98707
-    }
+      final String githubToken = Platform.environment['AUTOSUBMIT_TOKEN']!;
```

Comment:
Let's make this `GITHUB_TOKEN` (as it's more accurate)

Reply:
Done!